packages/http/package.json

```diff
@@ -28,6 +28,11 @@
     "./testing": {
       "types": "./dist/src/testing/index.d.ts",
       "default": "./dist/src/testing/index.js"
+    },
+    "./streams": {
+      "typespec": "./lib/streams/main.tsp",
+      "types": "./dist/src/streams/index.d.ts",
+      "default": "./dist/src/streams/index.js"
```

Comment:
I added a `default` entry to an empty file for 2 reasons:
1. Make the `bundler` happy (used by playground-website)
2. Prevent unforeseen issues with any JS bundlers out there. Based on the node.js documentation for conditional exports, it wasn't clear if `default` should always be specified or not. (Seems like it shouldn't but this felt like the safest path to take for now)